index.html

```diff
@@ -1,27 +1,32 @@
+<!DOCTYPE html>
 
-<html>
+<html lang="en">
 <head>
-	<meta charset="utf-8"><link rel="stylesheet" href="/main.css">	<title>Alia Borroho</title>
-	</head>
-	<body>
+    <meta charset="utf-8">
+
+    <title>Alia Borroho</title>
+    <link rel="stylesheet" href="/main.css">
+</head>
+<body>
+
 	<p>
-<img src="https://mail.google.com/mail/u/0/?ui=2&ik=326bbbf37a&view=fimg&th=14e433814216e904&attid=0.1&disp=inline&realattid=f_ibiz1aqf0&safe=1&attbid=ANGjdJ_4IwoHJMyjD0kY0nUMODfgzmagalIKQQd9a-V4W0qpglNT-qXetoLntsvpLgUqD1PIoMZL7uJk6iLe216XqidbUWUj5cT8QQwcThIoxErnHzH_2m0hmG6oSEw&ats=1435646996109&rm=14e433814216e904&zw&sz=w1252-h562">
-      </p>
+		<img src="http://bit.ly/1HCAJEB" alt="Alia">
```

Comment:
Even with this using a bitly link, Google mail profile images are not publicly. To see this, try opening the file in an ingognito browser window "CMD+SHIFT+N" in Chrome.

Things you may want to try to replace this with: Twitter profile images or images posted on a site like http://tinypic.com/